docs/verify/simulate-with-qiskit-primitives.mdx

```diff
@@ -71,36 +71,70 @@ estimator = Estimator()
 ### Run and get results
 
 Now that you have defined your `estimator`, you can run your estimation by calling the [`qiskit.primitives.Estimator.run`](../api/qiskit/qiskit.primitives.Estimator#run) method,
```

Comment:
change to statevector estimator, but note that BaseEstimatorV2 will have better docstring being the parent class.

```suggestion
Now that you have defined your `estimator`, you can run your estimation by calling the [`qiskit.primitives.StatevectorEstimator.run`](../api/qiskit/qiskit.primitives.StatevectorEstimator#run) method,
```